### PR TITLE
feature -> develop | Update advanced Section

### PR DESCRIPTION
**Pull request by wri-vikram-wri231**
Sir, I have update advanced section and fixed all issues and added permission in all pages of advanced section

I have checked all the functionality and attach video link

http://img-v3.resolved.io/screenshot/kyWlRnCkgg.mp4

### CODE REVIEW

File: Entities/FailedJob.php [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        return $query->select( array_diff( $this->getTableColumns(),$columns) );
+    }
+
+    //-------------------------------------------------
+    public function scopeBetweenDates($query, $from, $to)
+    {
+
+        if($from)
+        {
+            $from = \Illuminate\Support\Carbon::parse($from)
```
Comment (themodernpk) [inline Entities/FailedJob.php:58]: @wri-vikram-wri231 just use `Carbon`

File: Entities/Notification.php [hunk 2]
```diff
@@ -261,6 +261,8 @@ public static function postStore($request)
     public static function dispatch(Notification $notification, User $user, $inputs, $priority='default')
     {
 
+        $response = self::addInQueue($notification, $user, $inputs, $priority);
+        die("<hr/>line number=123");
```
Comment (themodernpk) [inline Entities/Notification.php:265]: @wri-vikram-wri231 remove this die

File: Vue/pages/advanced/batches/List.vue [hunk 3]
```diff
@@ -142,65 +125,60 @@
...
+                                <div class="level-left"></div>
 
                                 <div class="level-right">
-
-                                    <div class="level-item">
+                                    <div class="level-item ">
 
                                         <b-field>
+
                                             <b-datepicker
```
Comment (themodernpk) [inline Vue/pages/advanced/batches/List.vue:135]: @wri-vikram-wri231 `dusk` attributes are missing

Comment (themodernpk) [reply Vue/pages/advanced/batches/List.vue:135]: This feedback is for all form inputs and links

File: Vue/pages/advanced/batches/partials/ListLargeView.vue [hunk 4]
```diff
@@ -9,47 +9,61 @@
...
+                              @click="showModal(props.row)"
+                              type="is-default"
+                              rounded
+                              icon-left="eye">
+                        View
+                    </b-button>
                 </b-table-column>
 
-                <b-table-column field="failed_job_ids" label="Failed Job Ids" v-slot="props">
+                <b-table-column field="failed_job_ids" label="Failed Job Ids" width="12%" v-slot="props">
```
Comment (themodernpk) [inline Vue/pages/advanced/batches/partials/ListLargeView.vue:51]: @wri-vikram-wri231 don't define `width` in `%`. Give fixed value.

Comment (themodernpk) [reply Vue/pages/advanced/batches/partials/ListLargeView.vue:51]: Same feedback for all other sections too

**Review by themodernpk — CHANGES_REQUESTED** [review]

**Comment by wri-vikram-wri231** [comment]
@themodernpk Sir,
I have update all changes 
https://prnt.sc/xdmcgr
https://prnt.sc/xdmemh

**Pull request merged**